subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftPlatformToolProvider.java

```diff
@@ -107,7 +107,7 @@ public CommandLineToolSearchResult locateTool(ToolType compilerType) {
     @Override
     protected Compiler<StaticLibraryArchiverSpec> createStaticLibraryArchiver() {
         CommandLineToolConfigurationInternal staticLibArchiverTool = (CommandLineToolConfigurationInternal) toolRegistry.getStaticLibArchiver();
-        return new ArStaticLibraryArchiver(buildOperationExecutor, commandLineTool(ToolType.STATIC_LIB_ARCHIVER, "ar"), context(staticLibArchiverTool), workerLeaseService);
+        return new ArStaticLibraryArchiver(buildOperationExecutor, commandLineTool(ToolType.STATIC_LIB_ARCHIVER, "ar"), context(staticLibArchiverTool), false, workerLeaseService);
```

Comment:
❓ I'm wondering if this is the correct value here. Should it be `true`? If it's the wrong value, then we should expect the test to be currently failing on Windows. If it doesn't fail, we would need to verify if the test properly demonstrate the issue.

Reply:
I followed your suggestion to leave Swift out of the scope of this issue. The test covers only the cpp library plugin.